### PR TITLE
Update jarvis.py

### PR DESCRIPTION
**Pull request by narayan954**
Just trying out these features :P

### CODE REVIEW

File: jarvis.py [hunk 1]
```diff
@@ -168,12 +168,7 @@ def takeCommand():
             print(jok)
             speak(jok)
 
-        elif 'quit' in query:
-            print("Sure Sir, Thanks for letting me help you. Have a good day.")
-            speak("sure sir, thanks for letting me help you. Have a good day.")
-            exit()
-
-        elif 'exit' in query:
+        elif 'quit' in query or 'exit' in query:
```
Comment (narayan954) [inline jarvis.py:171]: Just saved some lines of code :D

File: jarvis.py [hunk 2]
```diff
@@ -168,12 +168,7 @@ def takeCommand():
             print(jok)
             speak(jok)
 
-        elif 'quit' in query:
```
Comment (narayan954) [inline jarvis.py:171]: This and line 176 code block can be merged effectively

**Review by narayan954 — COMMENTED** [review]
My first code review too i guess #hehe

**Comment by Harshgupta0007** [comment]
nice catch

**Pull request merged**